### DISCUSSION TITLE
support https://github.com/jtroo/kanata

### DISCUSSION THREAD

**Discussion by hab25** [manna-harbour/miryoku#150] [Q&A] [answered]
It is more [actively developed](https://www.libhunt.com/compare-kmonad-vs-kanata?ref=compare#:~:text=7.9-,Activity,-8.9) and has [advantages](https://github.com/jtroo/kanata/blob/main/docs/kmonad_comparison.) over kmonad.

**Comment by manna-harbour** [comment, marked as answer]
Thanks for the suggestion!

I've added a list of candidates for other implementations at the bottom of https://github.com/manna-harbour/miryoku/issues/90.

Due to limited time, I would only add a new implementation where it would cover a whole new class of keyboards. In the case of kanata, Miryoku is already supported on Linux and Windows by Miryoku KMonad and also on Linux by [𝑥MK](https://github.com/manna-harbour/xmk).

Adding a new implementation is fairly easy, see for example https://github.com/manna-harbour/miryoku_babel/commit/bc5c15a5e685f8d8e3ba7ff4d421648e8fa49bdf adding support for KMonad to Miryoku Babel, plus the makefile and include files in Miryoku KMonad. But as any changes to Miryoku need to be supported in all implementations, there's also an ongoing maintenance cost.

If you're interested in making a Miryoku Kanata implementation I'd be happy to accept a PR for Miryoku Babel, but you'd need to keep the Miryoku Kanata implementation up to date with future changes.

**Reply by hab25** [reply]
I do have some interest in doing that, but am also limited on time right now; so maybe in the next year or two.

Thank you for your reply (and for Miryoku!).

**Comment by 22ispencer** [comment]
i had some time this morning and i got a (mostly) functional fork of miryoku_kmonad working [here](https://github.com/22ispencer/miryoku_kanata)
